documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,231 @@
+
+---
+**NOTE**
+
+OIDC defines the scope `offline_access` and it is used to get an access token to be used at the UserInfo endpoint, and also to obtain a request token to refresh this access token.
+
+---
+
+The OIDC rules for `offline_access` as defined in [OIDC Offline Access](https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccess) apply. 
+
+Copied to here unchanged from the OIDC standard for clarity: 
+
+>  When offline access is requested, a prompt parameter value of consent MUST be used unless other conditions for processing the request permitting offline access to the requested resources are in place. The OP MUST always obtain consent to returning a Refresh Token that enables offline access to the requested resources. A previously saved user consent is not always sufficient to grant offline access.
+
+> Upon receipt of a scope parameter containing the offline_access value, the Authorization Server:
+
+>    MUST ensure that the prompt parameter contains consent unless other conditions for processing the request permitting offline access to the requested resources are in place; unless one or both of these conditions are fulfilled, then it MUST ignore the offline_access request,
+
+>    MUST ignore the offline_access request unless the Client is using a response_type value that would result in an Authorization Code being returned,
+
+>    MUST explicitly receive or have consent for offline access when the registered application_type is web,
+
+>    SHOULD explicitly receive or have consent for offline access when the registered application_type is native.
+
+The OIDC section on [OIDC Using Refresh Tokens](https://openid.net/specs/openid-connect-core-1_0.html#RefreshTokens) is not changed by this document.
+
+The OIDC security and privacy considerations regarding offline access and refresh tokens apply e.g. on
+[Token lifetime](https://openid.net/specs/openid-connect-core-1_0.html#TokenLifetime) and [Offline Access Privacy](https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccessPrivacy).
+
+## Client-Initiated Backchannel Authentication Flow
+
+The [Client-Initiated Backchannel Authentication (CIBA)](https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html) flow is used to initiate an out-of-band authentication of the end-user.
+
+Communication with the Backchannel Authentication Endpoint MUST utilize TLS.
+
+CIBA allows the Client to get the authentication result in three ways: poll, ping, or push. This profile allows clients to use the *poll* mode. In the Poll mode, the authentication result is retrieved by the Client by polling the OP's token endpoint using the new grant type. 
+
+### Authentication Request
+
+This CAMARA document makes the login_hint parameter REQUIRED if CIBA is used in a CAMARA context.
```

Comment:
I would reconsider mandating this query param.

In Number Verification & SimSwap API services it is required to pass mobile phone number in resource requests to complete the flow:
[simSwap API](https://github.com/camaraproject/SimSwap/blob/ddc3cc10fbeaa693dff935e783a2ea3050c7daf4/code/API_definitions/sim_swap.yaml#L209)
[numberVerification API](https://github.com/camaraproject/NumberVerification/blob/1a4bf7267c96f90874a87dff79e05fa845a0b6f5/code/API_definitions/CAMARA/number_verification.yaml#L158)

OIDC spec keeps this query param as OPTIONAL - it makes sense as it gives sub-projects some freedom to utilize it.

On the other hand - for CIBA _login_hint_ has to be mandated :)

Therefore a question: Is this section describes _generic_ Authorization Request, that works for both _CIBA_ & _auth_code grant flow_ ? If yes - some more context should be given to this section.